compiler-core/src/type_/prelude.rs

```diff
@@ -13,13 +14,17 @@ const RESULT: &str = "Result";
 const STRING: &str = "String";
 const UTF_CODEPOINT: &str = "UtfCodepoint";
 
-// TODO: use "gleam" as the prelude module name, not ""
+pub const PRELUDE_MODULE_NAME: &str = "gleam";
+
+pub fn is_prelude_module(module: &SmolStr) -> bool {
```

Comment:
Let's make this &str so it works with all strings, rather than only references to SmolStrs
```suggestion
pub fn is_prelude_module(module: &str) -> bool {
```

Reply:
TIL about deref coercion :D Thought I would need to as `as_str` on all the call sites but nope.